server/middlewares/auth.js

```diff
@@ -0,0 +1,28 @@
+const jwt = require('jsonwebtoken');
+
+const admin = require('../database/models/admin');
+
+require('env2')('config.env');
+
+const isAuthorized = (req, res, next) => {
+  jwt.verify(req.cookies.applicant, process.env.SECRET_KEY, (err, token) => {
+    if (err) {
+      res.status(401).send({ statusCode: 401, error: 'you are un authorized' });
+    } else {
+      next();
+    }
+  });
+};
+
+const isAdmin = (req, res, next) => {
+  const { email } = req.body;
+  admin.findOne({ email }).then((data) => {
```

Comment:
I think that admin is not one all the time also you have to check JWT for admin too , so please change `isAdmin` to be more suitable.